app/releasemanager.cpp

```diff
@@ -20,7 +20,9 @@
 #include "releasemanager.h"
 #include "drivemanager.h"
 
-#include "isomd5/libcheckisomd5.h"
+extern "C" {
+#include <libcheckisomd5.h>
```

Comment:
Well, for instance, the C standard libraries do this. If it was a bad practice, I think it wouldn't be there.